packages/vitest/src/node/core.ts

```diff
@@ -738,13 +747,13 @@ export class Vitest {
     return files
   }
 
-  private async isTargetFile(id: string, source?: string): Promise<boolean> {
-    const relativeId = relative(this.config.dir || this.config.root, id)
-    if (mm.isMatch(relativeId, this.config.exclude))
+  private async isTargetFile(id: string, project: WorkspaceProject, source?: string): Promise<boolean> {
```

Comment:
I think it's better to just move the method to the `VitestProject` class maybe?

Reply:
Moved to `WorkspaceProject` - hope you meant that one.